icicle/backend/cpu/src/field/cpu_vec_ops.cpp

```diff
@@ -368,7 +368,7 @@ class VectorOpTask : public TaskBase
 public:
   T m_intermidiate_res;    // pointer to the output. Can be a vector or scalar pointer
   uint64_t m_idx_in_batch; // index in the batch. Used in intermediate res tasks
-};                         // class VectorOpTask
+}; // class VectorOpTask
```

Comment:
this so annoying but I think that mac and linux format it differently and this is why we face those issues all the time

Reply:
Maybe we need the CI to fix the code and not us?

Reply:
it will have to make a new commit but maybe it's possible